scripts/format_notebooks.py

```diff
@@ -181,6 +181,58 @@ def preprocess(self, nb, resources):
         return nb, resources
 
 
+class VersionCheckPreprocessor(preprocessors.Preprocessor):
+    metadata_tag = "VersionCheck"  # tag for added cells so that we can find them easily
+
+    try:
+        # determine the current stellargraph version by importing the library's version file
+        sys.path.append(
```

Comment:
Maybe instead of modifying `sys.path` this could just do what `setup.py` does, and eval the `version.py` file directly:

https://github.com/stellargraph/stellargraph/blob/e12c5bedb6d99f55da630ad9c59526955d608fd2/setup.py#L74-L77

Thoughts?

Reply:
Yes, I think this is more reliable if another stellargraph library is already on the python path 👍